Nishant/Day_12/postgres assignment/src/controller/class.ts

```diff
@@ -0,0 +1,82 @@
+import { Context } from "vm";
+
+import * as serviceclass from '../services/class';
+
+interface  class_schedule_data{
+    classid:number;
+    classno:number;
+    subj_id:number;
+    subj_name:string;
+    t_id:number;
+    t_fname:string;
+}
+
+interface class_student_data{
+    class_id:number;
+    studid:number
+}
+
+interface student_details_data{
+    student_id:number;
+    fname:string;
+}
+
+export async function getClass(ctx: Context){
+    try{
+        let [rows]: Array<{rows: class_schedule_data}>=[];
+        rows=await  serviceclass.get_class();
+            ctx.response.status = 200;
+          ctx.response.type = 'application/json';
+          ctx.body=rows.rows;
+        }catch(err){
+          ctx.response.status = 500;
+          ctx.response.type = 'text/html';
+          ctx.body = "internal server error"; 
+          return ;
+        }
+}
+
+export async function getStudentByClassId(ctx: Context){
+    try{
```

Comment:
can we format this file, we can use linting to do this, check how we can do this.